test/integration/simple_example/controls/gcloud.rb

```diff
@@ -0,0 +1,71 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id = attribute('project_id')
+name = attribute('job_name')
+region = attribute('region')
+df_job_state = attribute('df_job_state')
+df_job_id = attribute('df_job_id')
+df_job_state_2 = attribute('df_job_state_2')
+df_job_id_2 = attribute('df_job_id_2')
+bucket_name = attribute('bucket_name')
+
+control "gcloud" do
+  title "gcloud configuration"
+  describe command("gcloud --project=#{project_id} dataflow jobs list --format=json") do
+    its(:exit_status) { should eq 0 }
+    its(:stderr) { should eq '' }
+
+
+    let(:data) do
+      if subject.exit_status == 0
+        JSON.parse(subject.stdout)
+      else
+        {}
+      end
+    end
+
+    context "gcloud dataflow jobs list OUTPUT" do
+      describe "contains all successful and failed jobs for the target project" do
+        it "should include the newly created dataflow jobs' job_ids" do
+          expect(data).to include(
```

Comment:
Rather than making two assertions within a single test case, these can be combined:

```ruby
expect(data).to include(
  including("id" => df_job_id),
  including("id" => df_job_id_2)
)
```

Also note that string variables do not need to be interpolated in string literals if there is no other content to be included.

Reply:
Resolved in
f90030a